src/python_testing/TC_TestAttrAvail.py

```
@@ -110,7 +108,8 @@ async def test_TC_TestAttrAvail(self):
         self.print_step("Operational Resume Command available ", cmd_should_be_there)
 
         self.step(4)
-        feat_should_be_there = await self.feature_guard(endpoint=self.endpoint, cluster=Clusters.BooleanStateConfiguration, feature_int=Clusters.BooleanStateConfiguration.Bitmaps.Feature.kAudible)
+        feat_should_be_there = await self.attribute_guard(endpoint=self.endpoint, cluster=Clusters.BooleanStateConfiguration,
+                                                  feature_int=Clusters.BooleanStateConfiguration.Bitmaps.Feature.kAudible)
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)